nservicebus/azure/data-distribution_content_[5.2,6).partial.md

```diff
@@ -0,0 +1,7 @@
+snippet: UniqueQueuePerEndpointInstance
```

Comment:
Hmm it should work. Looking at `ArticleRenderer` it applies the partials and then processes snippets so snippets inside partials should work fine...